### DISCUSSION TITLE
I CAN'T JOIN THE DISCORD CHANNEL

### DISCUSSION THREAD

**Discussion by thekingslee** [mrdbourke/zero-to-mastery-ml#78] [Q&A] [answered]
At section one, lecture three, there is a link to join the Community Discord. When I click on the link, it loads and loads forever, non stop. How can I join the community now? I really love the idea of using the community as an accountability tool.

1. I am stuck at Section 1, lecture 3
2. My problem is that I can'y join the Community Discord
3. __ __
4. I have tried the link provided on that lecture but it loads forever

**Comment by aman-wnq** [comment, marked as answer]
Make sure you are using the same email as Udemy on Discord as well.

**Comment by thekingslee** [comment]
Thank you.

**Comment by alsaagh** [comment]
you can join the discord  by clicking the following link <edited out, please see ZTM courses for the link>

**Reply by mrdbourke** [reply]
Hey @alsaagh thank you for offering to help, however, we keep the Discord link reserved for students of ZTM, preventing potential spam. I just edited the comment so the link is not public.

**Comment by SantiagoValencia77** [comment]
For some reason i joined the server today, but when i see now after hours later from enrolling. I'm no longer in the server i don't know why. When i try to log back in it doesn't let me :/ can someone please help. I had a few people working as a team and i can't get in

**Comment by davvyy000** [comment]
I can't join the discord either getting a 403 error, using the same email @mrdbourke

**Reply by Shanur880** [reply]
Try now, or close and reopen browser

**Comment by ghost** [comment]
I  can't join the discord channel,the link has been expired,
Is there any other ways to get into the discord

**Reply by kRiShNa-429407** [reply]
From where you purchased the course?
ZTM or Udemy??

**Comment by ghost** [comment]
I purchased it from udemy

**Reply by kRiShNa-429407** [reply]
Sorry to say but the link is only reserved for ZTM students

**Comment by ghost** [comment]
Ok sir,Thank you for your time.

**Comment by roma-sudo** [comment]
Hello, I am trying to "Accept the invite" to the Discord channel that is given in Udemy, it sends me to a new page but there it says that "Unable to accept invite" once I have clicked on the button. (screenshot attached). Thanks!
![Screenshot 2025-01-16 143855](https://github.com/user-attachments/assets/a3da4cbe-e14c-4027-91ce-85e8819c739e)

**Comment by othmar-alejandro** [comment]
Trouble accessing the discord channel. I got same email in udemy as in discord!

<img width="880" alt="Screenshot 2025-01-28 at 9 47 06 PM" src="https://github.com/user-attachments/assets/956f4283-acfa-4c3c-b99c-3d996bb39672" />

**Reply by joshp25-max** [reply]
I am still finding it difficult to access the discord channel. How do I go about it?

**Comment by ravi8jain** [comment]
Thanks
